.github/workflows/nightly.yaml

```diff
@@ -31,28 +31,6 @@ jobs:
           S3_BUILD_CACHE_SECRET_KEY: ${{ secrets.S3_BUILD_CACHE_SECRET_KEY }}
         run: ./gradlew build --stacktrace -x :smoke-tests:test
 
-  example-distro:
```

Comment:
This job does not need to populate remote cache. So it can run only on no-cache workflow.